Source/Core/Core/HW/WiimoteEmu/MotionPlus.h

```diff
@@ -90,12 +92,12 @@ struct MotionPlus : public Extension
 
     // address 0xF7
     // Games read this value to know when the data at 0x50 is ready.
-    // Value is 0x02 upon activation.
-    // Real M+ changes this value from 0x4, 0x8, 0xc, and finally 0xe.
+    // Value is 0x02 upon activation. (via a write to 0xfe)
+    // Real M+ changes this value to 0x4, 0x8, 0xc, and finally 0xe.
     // Games then trigger a 2nd stage via a write to 0xf1.
     // Real M+ changes this value to 0x14, 0x18, and finally 0x1a.
     // Note: We don't progress like this. We jump to the final value as soon as possible.
-    u8 challenge_progress;
+    u8 challenge_state;
```

Comment:
Is there a reason why this isn't an `enum class`?

Reply:
I guess not. I'll change it.